common/metrics/defs.go

```diff
@@ -824,6 +825,7 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		PersistenceErrConditionFailedCounter:          {metricName: "persistence.errors.condition-failed", metricType: Counter},
 		PersistenceErrTimeoutCounter:                  {metricName: "persistence.errors.timeout", metricType: Counter},
 		PersistenceErrBusyCounter:                     {metricName: "persistence.errors.busy", metricType: Counter},
+		PersistenceErrSampledCounter:                  {metricName: "persistence.errors.sampled", metricType: Counter},
```

Comment:
I don't think we should use "errors" in the name of the counter.  Let's just call the counter persistence.sampled.